modal/sandbox.py

```diff
@@ -325,19 +326,27 @@ async def exec(self, *cmds: str, pty_info: Optional[api_pb2.PTYInfo] = None):
 
     @property
     def stdout(self) -> _StreamReader:
-        """`StreamReader` for the sandbox's stdout stream."""
+        """
+        [`StreamReader`](docs/reference/modal.io_streams#modalio_streamsstreamreader) for
```

Comment:
I prefer single-click for these things rather than having to go to the search box and type in the name.